pgmpy/factors/ContinuousNode.py

```diff
@@ -0,0 +1,230 @@
+from scipy import integrate
+from scipy.stats import rv_continuous
+
+import numpy as np
+
+
+class ContinuousNode(rv_continuous):
+    """
+    Class for continuous node representation. This is a subclass of
+    scipy.stats.rv_continuous.
+    This allows representation of user defined continuous distribution
+    by specifying a function to compute the probability density function
+    of the distribution.
+    All methods of the scipy.stats.rv_continuous class can be used on
+    the objects.
+    This supports an extra method to discretize the continuous distribution
+    into a discrete factor using various methods.
+    """
+    def __init__(self, pdf, lb=None, ub=None):
+        """
+        Parameters
+        ----------
+        pdf : function
+        The user defined probability density function.
```

Comment:
@yashu-seth Check out the formatting in other docstrings. Add a few extra spaces on the description line.